### PR TITLE
US 95

### PR DESCRIPTION
**Pull request by yuupiaa**
Finished US-95. Ready to merge into DevB. Let me know what bugs to fix before I do.

### CODE REVIEW

File: src/main/java/memoranda/driver/DriverObject.java [hunk 1]
```diff
@@ -1,148 +1,181 @@
+/*
+  File:Bus.java
```
Comment (BrittneePacini) [inline src/main/java/memoranda/driver/DriverObject.java]: Remember to update the file names 😄

File: src/main/java/memoranda/ui/EventDialogDriver.java [hunk 2]
```diff
@@ -239,14 +262,16 @@ public void actionPerformed(ActionEvent e) {
...
 
     
-
+//deleted code below??
+/*
     void okB_actionPerformed(ActionEvent e) {
     	Object sourceObject = e.getSource();
     	addDriverB();
     	
     	//what is .dispose()????
```
Comment (BrittneePacini) [inline src/main/java/memoranda/ui/EventDialogDriver.java]: dispose closes the box

Comment (BrittneePacini) [inline src/main/java/memoranda/ui/EventDialogDriver.java]: Dispose closes the box

**Comment by BrandenTurley** [comment]
Looked good

**Pull request merged**